packages/tooltip/stories/tooltip.stories.ts

```diff
@@ -104,3 +103,54 @@ export const wIcon = (): TemplateResult => {
 wIcon.story = {
     name: 'w/ Icon',
 };
+
+const overlayStyles = html`
+    <style>
+        html,
+        body,
+        #root,
+        #root-inner,
+        sp-story-decorator {
+            height: 100%;
+            margin: 0;
+        }
+
+        sp-story-decorator::part(container) {
```

Comment:
This is nice to see someone leverage!

Reply:
tbh I didn't look at it very closely, I just noticed that Overlay was doing what I needed so I copy-pasted the styles!

Reply:
OK, so you're saying I'm congratulating myself for using the thing I made for myself 😜

Reply:
I mean, it worked! :tada: